api/utils/http.go

```diff
@@ -36,6 +38,17 @@ func BadRequest(cause error) error {
 	}
 }
 
+func StringToBoolean(boolStr string, defaultVal bool) (bool, error) {
+	if boolStr == "" {
+		return defaultVal, nil
+	} else if boolStr == "false" {
+		return false, nil
+	} else if boolStr == "true" {
```

Comment:
Given it's a direct comparison, would it be useful to use string trim ?

Reply:
Yeah, it might be helpful, will add that

Reply:
I just checked I don't think other endpoints use trimming for parameters, not sure now if there has to be this inconsistency